python/activity_stream/activity_stream.py

```diff
@@ -613,6 +613,10 @@ def show_new_note_dialog(self, modal=True):
 
         :param bool modal: Whether the dialog should be shown modally or not.
         """
+        if self._entity_id == None:
```

Comment:
nitpick: `if self._entity_id is None:` is the idiom we use in the rest of the code base. See [PEP-8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):

_Comparisons to singletons like None should always be done with is or is not , never the equality operators._